.github/workflows/update-ghc.yaml

```diff
@@ -0,0 +1,37 @@
+name: Update GHC
+on:
+  schedule:
+    # run weekly on Thursday
+    - cron:  '0 15 * * THU'
+  workflow_dispatch: # allows manual triggering
+
+jobs:
+  update_ghc:
+    name: GHC ${{ matrix.ghc }} Update
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ghc:
+          - '9.2'
+          - '9.4'
+          - '9.6'
```

Comment:
Just to confirm my understanding. Why do we loop over major-minor versions via a matrix instead of doing that within the update script with a single action run? Is it so that if multiple updates fall onto the same run we still get separate PRs per major-minor version?

Reply:
Yes, the reason was that I wanted to have separate PRs for each stable GHC major.minor version.